pkgs/by-name/sa/satisfactorymodmanager/package.nix

```diff
@@ -0,0 +1,93 @@
+{
+  lib,
+  fetchurl,
+  buildGoModule,
+  fetchFromGitHub,
+  nodejs,
+  pnpm_8,
+  wails,
+  wrapGAppsHook3,
+}:
+
+let
+  # is this reproducible?
+  fonts-declarations = fetchurl {
+    name = "satisfactorymodmanager-font-declarations.css";
+    url = "https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible&family=Roboto&family=Roboto+Mono&family=Material+Icons&family=Noto+Color+Emoji&display=swap";
+    hash = "sha256-h0y8wHmpYbJsd3ywqHuLyGM8oeqY59QwOUcFgGnMZ+Q=";
+  };
```

Comment:
The licenses of this should be added. The `Robot-Mono` font is `ASL2` licensed for example.

Reply:
The files are generated by querying a tolgee localisation server. (pnpm run translations).
AFAICT the API key included in the repo doesn't have permisssion to fetch a specific revision of the translations, only the latest. This is why I vendored the files. Might need to contact upstream about this.